renderdoc/driver/vulkan/wrappers/vk_queue_funcs.cpp

```diff
@@ -1006,6 +1010,9 @@ void WrappedVulkan::CaptureQueueSubmit(VkQueue queue,
           it != record->bakedCommands->cmdInfo->sparse.end(); ++it)
         GetResourceManager()->MarkSparseMapReferenced(*it);
 
+      accelerationStructures.insert(accelerationStructures.size(),
+                                    record->bakedCommands->cmdInfo->accelerationStructures);
```

Comment:
This is only being processed while in `capframe` - i.e actually capturing a frame. This code will need to run unconditionally while capturing to ensure we have up to date information about when the AS is built when we do start capturing a frame.